### PR TITLE
Update discussion messages UI

### PR DESCRIPTION
**Pull request by felipecalvo**
## :dart: Goal

Improve the forum messages UI.

## :memo: Details

There's a visual fix and a couple of unrequested changes, which I'll detail in subsequent comments. Please provide your feedback :smile:

### Before
Screenshot from 2021-02-09 01-39-47

### After
image

### CODE REVIEW

**Comment by felipecalvo** [comment]
### Action icons

Icons are larger and more spread out. 'Not a question' icon is clearer.

Screenshot_2021-02-09

Screenshot_2021-02-09 owner owner - Aprendé a programar

**Comment by felipecalvo** [comment]
### Moderator badge

Star (and its tooltip) is replaced with a badge. Here are a couple alternatives. I prefer the solid gold or solid pink but any of the four should be fine.

Screenshot_2021-02-09 Franco Bulgarelli - Aprendé a programar(1)
Screenshot_2021-02-09 Franco Bulgarelli - Aprendé a programar(4)
Screenshot_2021-02-09 Franco Bulgarelli - Aprendé a programar(2)
Screenshot_2021-02-09 Franco Bulgarelli - Aprendé a programar(3)

**Comment by felipecalvo** [comment]
### Female moderators

Whatever color or design we choose, we can now properly credit the women behind the screen :woman_technologist: 

Screenshot_2021-02-09 Franco Bulgarelli - Aprendé a programar

**Comment by felipecalvo** [comment]
### Hover on action buttons

They change colors when hovering with the mouse to give a little more feedback.
icon

File: app/helpers/discussions_helper.rb [hunk 1]
```diff
@@ -195,4 +195,8 @@ def discussion_filter_params_without_page
   def should_show_approved_for?(user, message)
     !user&.moderator_here? && message.approved? && !message.from_moderator?
   end
+
+  def gendered_moderator(user)
+    user.female? ? t(:female_moderator) : t(:moderator)
+  end
```
Comment (felipecalvo) [inline app/helpers/discussions_helper.rb:201]: I'm aware this isn't a good idea on a bigger scale, but it's our first and only (so far) gender-aware translation. We should check out [i18n-inflector](https://github.com/siefca/i18n-inflector) if this necessity grows.

**Comment by felipecalvo** [comment]
Finally we're going for a genderless `Moderation` badge - `Mentoría` in Spanish - so we're not trapped into binary pitfalls :+1: 

image

**Review by luchotc — APPROVED** [review]
This is so great @felipecalvo :clap: 
Just one minor detail, I think `moderator` translation is actually used, so could we please keep it before merging this one?

**Comment by luchotc** [comment]
Nvm, I'll just add them :)

**Comment by felipecalvo** [comment]
I don't see it used at all, am I missing something? The only uses of `:moderator` are related to FactoryBot and permissions.

**Pull request merged**